src/mysql.erl

```diff
@@ -437,6 +438,49 @@ execute_transaction(Conn, Fun, Args, Retries) ->
             {aborted, Aborted}
     end.
 
+-spec change_user(Conn, Username, Password) -> Result
+    when Conn :: connection(),
+         Username :: iodata(),
+         Password :: iodata(),
+         Result :: ok | {error, server_reason}.
+change_user(Conn, Username, Password) ->
+    gen_server:call(Conn, {change_user, Username, Password, undefined}).
+
+-spec change_user(Conn, Username, Password, Options) -> Result
+    when Conn :: connection(),
+         Username :: iodata(),
+         Password :: iodata(),
+         Options :: [Option],
+         Result :: ok | {error, server_reason},
```

Comment:
`server_reason()`

Reply:
Copied over from the previous section, will fix.